bindata/network/openshift-sdn/003-rbac-controller.yaml

```diff
@@ -0,0 +1,160 @@
+# Only for testing against vanilla k8s
+# will go away when running against the real installer
```

Comment:
"will go away" because the namespace will already be created by someone else, right? The comment makes it sound like we don't need the namespace when using the real installer...

Reply:
Turns out it won't go away; updated the comment.